markdown/events/2021/bytesize-22-nf-core-eager

```diff
@@ -0,0 +1,25 @@
+---
+title: "Bytesize 22: nf-core/eager"
+subtitle: James Fellows Yates - MPI-EVA, Germany
+type: talk
+start_date: "2021-10-05"
+start_time: "13:00 CEST"
+end_date: "2021-10-05"
+end_time: "13:30 CEST"
+youtube_embed: https://youtu.be/fObuLeGhQRo
+location_url:
+ - https://youtu.be/fObuLeGhQRo
+ - https://zoom.us/j/95310380847
+---
+
+# nf-core/bytesize
+
+Join us for a special pipeline-focussed episode of our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize 22: nf-core/eager
+
+This week, James Fellows Yates ([@jfy133](https://github.com/jfy133/)) will tell us all about the nf-core/eager pipeline.
```

Comment:
```suggestion
This week, James Fellows Yates ([@jfy133](https://github.com/jfy133/)) will tell us all about the nf-core/eager pipeline.

nf-core/eager is one of the largest and/or most complex nf-core pipelines. It is a best practise (meta)genomics primarily aimed at (but not limited to) processing ancient DNA. This talk will give an overview of the pipeline, discuss the impact of the particularlities of ancient DNA, and some of the challenges and solutions encountered when writing this pipeline.
```

Reply:
@jfy133 I've added bullet points to maintain a similar style to previous talks.